addons/survey/views/survey_templates.xml

```diff
@@ -13,6 +16,12 @@
         <xpath expr="//header" position="after">
             <div id="wrap" class="oe_structure oe_empty"/>
         </xpath>
+        <xpath expr="//header" position="attributes">
+            <attribute name="class" add="o_survey_header" separator=" "/>
+        </xpath>
+        <xpath expr="//main" position="attributes">
+            <attribute name="class">o_survey_main</attribute>
```

Comment:
isn't it possible to move that in a real survey template ? Normally you should never touch the main laout attributes and keep everything contained within the addon specific template, here survey_page_fill (div class="wrap o_survey_page_fill" for example).